tools/docker-compose/supervisor.conf

```diff
@@ -4,7 +4,7 @@ minfds = 4096
 nodaemon=true
 
 [program:celeryd]
-command = celery worker -A awx -l DEBUG -B -Ofair --autoscale=100,4 --schedule=/celerybeat-schedule -Q tower_scheduler,tower_broadcast_all,%(ENV_AWX_GROUP_QUEUES)s,%(ENV_HOSTNAME)s -n celery@%(ENV_HOSTNAME)s
+command = celery worker -A awx -l DEBUG -B --autoscale=20,3 -Ofair -s /var/lib/awx/beat.db -Q tower_broadcast_all -n celery@%(ENV_HOSTNAME)s
```

Comment:
Is the change to `--autoscale` intentional here?  What's the rationale?

Reply:
This change only affects our development tooling. Our dev environments don't need to scale that much and don't need 4 hot celery tasks waiting.

Unless you are running this on a system larger than my workstation at home that is ;)